src/Bridge/Doctrine/Repositories/FillableRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Webhooks\Bridge\Doctrine\Repositories;
+
+use EoneoPay\Externals\ORM\Repository;
+use EoneoPay\Webhooks\Bridge\Doctrine\Repositories\Interfaces\FillableRepositoryInterface;
+
+class FillableRepository extends Repository implements FillableRepositoryInterface
```

Comment:
final?

Reply:
Any entity which wants fillable can extend this class hence no final.

Reply:
It should be abstract if it isnt final

Reply:
**Resolution:**

`final` it is